src/test/resources/TestUtils.java

```diff
@@ -657,6 +657,8 @@ public static void startServer() {
     app.post("/submit-bug", bugController.submitBug);
     app.post("/find-bug", bugController.findBug);
     app.post("/organization-sign-up", orgController.enrollOrganization(securityUtils));
+    app.post("/dashboard", orgController.dashboard);
+    app.post("/all-orgs", orgController.listOrgs);
```

Comment:
this is a bit more of a system design thing, but instead of calling it a /dashboard route (which means that any query for getting number of clients, e.g.) can you break it down into like this route:
POST /get-usertype-count
parameters: <Optional> usertype[], <Optional> orgNames[],
return <Optional<Integer>> userCount[]
basically, you could hit the /get-user-count route with usertype as [clients, workers, admins], and orgNames[orgNameHere], which would query the number of clients and workers within the organization name that we specified (for example broad street ministries)
so an example request would be like
{userType: [User.WORKER ], orgNames: ["broad street ministries"]}, which would return userCount: 
{ User.WORKER: 20 } (aka 20 users in BSM)
 or another one would be like 
{userType: [User.CLIENT, User.ADMIN], orgNames: ["broad street ministries", "EndHomelesseness"]}, which would return userCount: 
{ User.CLIENT: 100,  User.WORKER: 30 } which would be 100 clients and 30 workers in both BSM and end homelessness.

The last case if if orgNames is missing/null, which would mean that it would return the total number of clients/workers/etc.